stock-filters/ChirpVillage/BiomeSettings.py

```diff
@@ -0,0 +1,17 @@
+# biome names come from biome_types.py
+# material names are currently hardcoded
+# block id's can be found at https://minecraft-ids.grahamedgecombe.com/
+biomeSettings = {
+	"Plains": { # Default
```

Comment:
This is a little nitpicky so I'll let you decide if you want to change it or not. It might be easier to read if we just have wall/fence/floor as opposed to wall_material/fence_material and then change biomeSettings to materials reading materials['Plains']['floor'] gives the same information with less text

Reply:
Good idea